modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/ObjectEntryResourceTest.java

```diff
@@ -5774,6 +5778,48 @@ public void testGetObjectEntriesFilteredBySystemDate() throws Exception {
 		}
 	}
 
+	@Test
+	public void testGetObjectEntriesVersionsPage() throws Exception {
+		_objectDefinition1.setEnableObjectEntryVersioning(true);
+		_objectDefinition2.setEnableObjectEntryVersioning(true);
+
+		_objectDefinition1 =
+			_objectDefinitionLocalService.updateObjectDefinition(
+				_objectDefinition1);
+		_objectDefinition2 =
+			_objectDefinitionLocalService.updateObjectDefinition(
+				_objectDefinition2);
+
+		_objectEntry1 = ObjectEntryTestUtil.addObjectEntry(
+			_objectDefinition1, _OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_1);
+
+		_objectEntry2 = _objectEntryLocalService.updateObjectEntry(
```

Comment:
@vngoliveira you don't need this `_objectEntry2`. 

- Reuse `_objectEntry1` or do not need to assign to any variable. 

In this case, please create a new local variable:

```java
ObjectEntry objectEntry = ObjectEntryTestUtil.addObjectEntry(
      _objectDefinition1, _OBJECT_FIELD_NAME_1, _OBJECT_FIELD_VALUE_1);

_objectEntryLocalService.updateObjectEntry(
     TestPropsValues.getUserId(), objectEntry.getObjectEntryId(),
....
```